pkg/tools/backplanecli/backplanecli.go

```diff
@@ -36,42 +35,17 @@ func (t *Tool) Install() error {
 		return err
 	}
 
-	// Determine which assets to download
-	var checksumAsset *gogithub.ReleaseAsset
-	var backplaneArchiveAsset *gogithub.ReleaseAsset
-	arch := runtime.GOARCH
-	if arch == "amd64" {
-		arch = "x86_64"
+	matches := github.FindAssetsForArchAndOS(release.Assets)
+	if len(matches) != 1 {
+		return fmt.Errorf("unexpected number of assets found matching system spec: expected 1, got %d.\nMatching assets: %v", len(matches), matches)
 	}
-	for _, asset := range release.Assets {
-		if asset.GetName() == "checksums.txt" {
-			if checksumAsset.GetName() != "" {
-				return fmt.Errorf("detected duplicate backplane-cli checksum assets")
-			}
-			checksumAsset = asset
-			continue
-		}
-		// Exclude assets that do not match system architecture
-		if !strings.Contains(asset.GetName(), arch) {
-			continue
-		}
-		// Exclude assets that do not match system OS
-		if !strings.Contains(strings.ToLower(asset.GetName()), strings.ToLower(runtime.GOOS)) {
-			continue
-		}
+	backplaneArchiveAsset := matches[0]
```

Comment:
Is there ever a case where more than one match is valid?  Should that logic be in FindAssetsForArchAndOS similar to how FindAssetsMatching checks for 0 length slices?